### ISSUE TITLE
add a seperate dlfcn-win32 package?

### ISSUE THREAD

**Issue by kitech** [Martchus/PKGBUILDs#55]
I need only the package some case, could you add a standalone package to repo?

**Comment by Martchus** [comment]
So you want me to add the following AUR package to my binary repository?  
https://aur.archlinux.org/packages/mingw-w64-dlfcn

**Comment by kitech** [comment]
yes, dlfcn for mingw package, put in your archlinux binary repo

**Comment by kitech** [comment]
another thing, qt office mingw package using mingw gcc dw2, ownsuff repo using gcc sjlj.
what do you think about that.

**Comment by Martchus** [comment]
Ok, I've added the package. But it didn't take long to compile anyways.

---

About the exception handling, I'm also thinking about that - especially since MSYS2 mingw-w64 packages also do it differently.  
Since this involves rebuilding all packages, I'll not change the exception handling very soon and for me there are still open questions and things I'm not sure about. But I've created an [issue](https://github.com/Martchus/PKGBUILDs/issues/56).
